components/input/TextArea.tsx

```diff
@@ -9,15 +9,21 @@ import { fixControlledValue, resolveOnChange } from './Input';
 export interface TextAreaProps extends RcTextAreaProps {
   allowClear?: boolean;
   bordered?: boolean;
+  hasCount?: boolean;
 }
 
 export interface TextAreaState {
   value: any;
 }
 
+function countSymbols(text = ''): number {
+  if (text == null || text == undefined) return 0;
```

Comment:
`if (!text) return 0;` is enough, there is no way for `text === undefined`

Reply:
thanks,what should I do, only keep typeof text ===’null‘ ?

Reply:
that is ok